src/dbg/debugger_commands.cpp

```diff
@@ -1997,6 +2002,41 @@ CMDRESULT cbDebugKillthread(int argc, char* argv[])
     return STATUS_ERROR;
 }
 
+CMDRESULT cbDebugCreatethread(int argc, char* argv[])
+{
+    if(argc < 2)
+        return STATUS_ERROR;
+    duint Entry = 0;
+    duint Argument = 0;
+    if(!valfromstring(argv[1], &Entry))
+        return STATUS_ERROR;
+    if(!MemIsCodePage(Entry, false))
+        return STATUS_ERROR;
+    if(argc > 2)
+    {
+        if(!valfromstring(argv[2], &Argument))
+            return STATUS_ERROR;
+    }
+    DWORD ThreadId = 0;
+    if(ThreaderCreateRemoteThread(Entry, true, reinterpret_cast<LPVOID>(Argument), &ThreadId) != 0)

```

Comment:
The debugging engine can use this information to improve `ThreaderIsAnyThreadActive` implementation so it won't return false when a new thread is about to be created.